webdriver/tests/support/helpers.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import print_function
+import sys
+
+import webdriver
+
+from tests.support import defaults
+
+def ignore_exceptions(f):
+    def inner(*args, **kwargs):
+        try:
+            return f(*args, **kwargs)
+        except webdriver.error.WebDriverException as e:
+            print("Ignored exception %s" % e, file=sys.stderr)
```

Comment:
I don't understand the linter failure:
> ERROR:lint:webdriver/tests/support/helpers.py:13: Print function used (PRINT STATEMENT)

You are already using the print function, or do I miss something?

Reply:
This code is not new.

Reply:
Correct. That's why I don't understand why the lint job is failing on it.

Reply:
According to the docs (https://web-platform-tests.org/writing-tests/lint-tool.html),

PRINT STATEMENT: A server-side python support file contains a print statement; fix: remove the print statement or replace it with something else that achieves the intended effect (e.g., a logging call).


So, we need to figure out the correct logger to use here. I'm not really familiar enough to know the answer. @gsnedders ?

Reply:
Actually, this call to print was whitelisted for the linter at https://github.com/web-platform-tests/wpt/blob/master/lint.whitelist#L103. Now that you moved it you also want to update its location.

Reply:
Got it!